WEBEUFLEX-Dimos Christidis-Yusuf Abdulkarim- WEEK1-DAY4

OldDognoob opened the pull request.

OldDognoob committed:
no time for big changes

OldDognoob committed:
title position

OldDognoob committed:
html changes-css changes

OldDognoob committed:
adding padding-marging-width-felx-justify

OldDognoob committed:
footer changes

OldDognoob committed:
adding changes in css

OldDognoob committed:
Dimos-Christidis

Becheru888 reviewed (changes requested):
Please finish the project!

OldDognoob committed:
index.css 

Initial changes to the first and second page. Refactor the code and do also the stretch goal.

OldDognoob committed:
index.html

index refactor code

OldDognoob committed:
service.html

refactor the code

OldDognoob reviewed (commented):
All pages have been changed. 
Refactored, media and flex properties utilize according to the MVP points.
Stretch goals attended at the best.

OldDognoob committed:
update service.html

A small gift, listen to it to your browser. Thank you & Good luck to all your dreams.
Send my regards to all of them.. Adios Amigos!!!!

OldDognoob reviewed (commented):
small changes to service.html ...
